src/services/tests/clock-test.ts

```diff
@@ -0,0 +1,43 @@
+/**
+ * @license
+ * Copyright 2019 Google LLC.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * Code distributed by Google as part of this project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+import {assert} from '../../platform/chai-web.js';
+import {Services} from '../../runtime/services.js';
+import {Instant} from '../../common/time/instant.js';
+import {TimeUnit} from '../../common/time/timeunit.js';
+import '../clock-service.js';
+
+describe('services: clock', () => {
+  it('gets a clock value', async () => {
+    // SECONDS
+    let clockValue = await Services.request({call: 'clock.now', timeUnit: 'SECONDS'});
+    assert.isDefined(clockValue);
+
+    let instant = Instant.fromString(clockValue);
```

Comment:
Will instant be importable / used from within particles as well?

Reply:
yes, it will be added to the default environment via loader.  PRs to follow.